### PR TITLE
:sparkles: new partners section added

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206412083694655

### CODE REVIEW

File: public/images/partners/gotbit.png
Comment (MarioBatalha) [inline public/images/partners/gotbit.png:1]: Image format must be svg or webp.

File: public/images/partners/stoned-ape-crew.png
Comment (MarioBatalha) [inline public/images/partners/stoned-ape-crew.png:1]: Image format must be svg or webp.

File: public/images/partners/zebec.png
Comment (MarioBatalha) [inline public/images/partners/zebec.png:1]: Image format must be svg or webp.

File: views/home/partners/title.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      variant="medium"
+      fontFamily="Asap"
+      fontSize="3.5rem"
+      textAlign="center"
+      letterSpacing="-0.0156rem"
+    >
+      Partners
+    </Typography>
+    <Box position="absolute" top="-5rem" width="9rem" right="-2rem">
+      <HandsSVG maxHeight="100%" maxWidth="100%" width="100%" height="60%" />
```
Comment (MarioBatalha) [inline views/home/partners/title.tsx:30]: Is missing hand-waving animation

File: public/images/partners/zebec.png
Comment (MarioBatalha) [inline public/images/partners/zebec.png:1]: Check if this image can be in svg or webp

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/home/partners/title.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      fontWeight="700"
+      variant="medium"
+      fontFamily="Asap"
+      fontSize="3.5rem"
+      textAlign="center"
+      letterSpacing="-0.0156rem"
+    >
+      Partners
+    </Typography>
+    <Motion
```
Comment (MarioBatalha) [inline views/home/partners/title.tsx:29]: It's not the right animation, look for waving hand animation.

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**